recipe/meta.yaml

```
@@ -66,7 +68,7 @@ outputs:
         - cmake
         - make
         - llvm-openmp  # [osx]
-        - libgomp      # [linux]
+        - libgomp      # [not osx]
       host:
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler != "None"]
         - nccl                                      # [linux and cuda_compiler != "None"]
```

Sounds like we will want this in `run` as well

```suggestion
        - libgomp                                   # [win]
```